src/widgets/gauge.rs

```diff
@@ -341,9 +349,36 @@ impl<'a> LineGauge<'a> {
     ///
     /// `style` accepts any type that is convertible to [`Style`] (e.g. [`Style`], [`Color`], or
     /// your own type that implements [`Into<Style>`]).
+    #[deprecated(
+        since = "0.27.0",
+        note = "You should use `LineGauge::filled_style and LineGauge::unfilled_style` instead."
```

Comment:
Backticks are missing:
```suggestion
        note = "You should use `LineGauge::filled_style` and `LineGauge::unfilled_style` instead."
```

Not sure about the phrasing as `filled_style` is the direct equivalent and unfilled is new? Maybe tell the user to replace with filled and consider the new unfilled?